server/service/settings/update.ts

```diff
@@ -0,0 +1,17 @@
+import { BaseProtocol, DB } from '@server/db'
+import { Settings } from '@core/meta/settings'
+import { SettingsRepository } from '@server/repository'
+
+export const update = async (
+  props: {
+    settings: Settings
+  },
+  client: BaseProtocol = DB
+): Promise<Settings> => {
+  const { settings } = props
+
+  return client.tx(async (t) => {
+    // TODO: Add activity log entry (public schema?)
```

Comment:
nope. we assume for now is always fra. or we can think about this later? we can remove the todo?